plugins/orchestrator/src/components/WorkflowDefinitionViewerPage/WorkflowDefinitionViewerPage.stories.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+
+import { TestApiProvider, wrapInTestApp } from '@backstage/test-utils';
+
+import { Meta, StoryObj } from '@storybook/react';
+
+import {
+  WorkflowItem,
+  WorkflowOverview,
+} from '@janus-idp/backstage-plugin-orchestrator-common';
+
+import { orchestratorApiRef } from '../../api';
+import { orchestratorRootRouteRef } from '../../routes';
+import { WorkflowDefinitionViewerPage } from './WorkflowDefinitionViewerPage';
+
+const mockWorkflowItem: WorkflowItem = {
+  definition: {
+    id: 'sample.workflow',
+    version: '1.0.0',
+    specVersion: '1.0.1',
+    name: 'Sample workflow',
+    states: [{}],
+  },
+  uri: 'sample.workflow.sw.yaml',
+};
```

Comment:
There are already some mocks in the `src/__fixtures__` directory. Please see consider reusing them or move those that are unique to that folder in order to keep this module leaner.